sdk/servicebus/Azure.Messaging.ServiceBus/src/Receiver/ServiceBusReceiverClient.cs

```diff
@@ -98,14 +92,6 @@ public abstract class ServiceBusReceiverClient : IAsyncDisposable
         ///
         internal TransportConsumer Consumer { get; }
```

Comment:
Not worth doing at this point with the design in flux, but once we establish a name for the consumer/receiver concept, we should make a sweep and align terms.